bigbluebutton-html5/private/config/settings.yml

```diff
@@ -240,6 +241,12 @@ public:
       dataSaving:
         viewParticipantsWebcams: true
         viewScreenshare: true
+      # Options that are sent to the transcription backed (only Gladia is supported for now)
+      transcription:
+        # Indicates if the transcription backend should include partial results
+        partialUtterances: true
+        # The minumum length (in seconds) an utterance has to have for we to use it
+        minUtteranceLength: 1
```

Comment:
Note: Setting it to `3` for Bulgarian improved the quality of the transcription significantly. I also combined it with disabling partialUtterances though.